ui/app/AppLayouts/Chat/panels/communities/CommunityMembersTabPanel.qml

```diff
@@ -0,0 +1,115 @@
+    signal unbanUserClicked(string id)
+
+    enum TabType {
+        AllMembers,
+        BannedMembers
+    }
+
+    property int panelType: CommunityMembersTabPanel.TabType.AllMembers
+
+    Layout.fillWidth: true
+    Layout.fillHeight: true
+
+    ColumnLayout {
+        anchors.fill: parent
+
+        StatusBaseInput {
+            id: memberSearch
+            Layout.preferredWidth: 351
+            rightPadding: 0
+            placeholderText: root.placeholderText
+            icon.name: "search"
+            enabled: model.count > 0
+        }
+
+        ListView {
+            id: membersList
+
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+
+            model: root.model
+            clip: true
+
+            delegate: StatusMemberListItem {
+                id: memberItem
+
+                readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
+                readonly property bool isHovered: !memberItem.itsMe && memberItem.sensor.containsMouse
+
+                components: [
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.AllMembers) && isHovered
+                        text: qsTr("Kick")
+                        type: StatusBaseButton.Type.Danger
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.kickUserClicked(model.pubKey, model.displayName)
+                    },
+
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.AllMembers) && isHovered
+                        text: qsTr("Ban")
+                        type: StatusBaseButton.Type.Danger
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.banUserClicked(model.pubKey, model.displayName)
+                    },
+
+                    StatusButton {
+                        visible: (root.panelType === CommunityMembersTabPanel.TabType.BannedMembers) && isHovered
+                        text: qsTr("Unban")
+                        type: StatusBaseButton.Type.Normal
+                        size: StatusBaseButton.Size.Small
+                        onClicked: root.unbanUserClicked(model.pubKey)
+                    }
+                ]
+
+                width: membersList.width
+                visible: memberSearch.text === "" || title.toLowerCase().includes(memberSearch.text.toLowerCase())
+                height: visible ? implicitHeight : 0
+                color: "transparent"
+
+                pubKey: Utils.getElidedCompressedPk(model.pubKey)
+                nickName: model.localNickname
+                userName: {
+                    let tempUserName = model.displayName
+                    if (memberItem.itsMe) {
+                        tempUserName += qsTr(" (You)")
+                    }
+                    return tempUserName
+                }
```

Comment:
(optional)
```suggestion
                userName: memberItem.itsMe ? model.displayName + qsTr(" (You)") : model.displayName
                }
```

Shouldn't this logic come from the backend? Not a blocker for this PR, just the trigger for the discussion.

Reply:
Yes, we have a displayName field and it could be placed there ... however, I am not sure if every view requires this information. If yes, then no problem.

Reply:
https://github.com/status-im/status-desktop/issues/6455

Reply:
@osmaczko Old display, I removed "You" here.

Reply:
Cool! I've created a global issue to get rid of "You" from everywhere. Mentioned it here to resolve the discussion.